### PR TITLE
A light angular 2+ sample

### PR DESCRIPTION
**Pull request by younos1986**
A complete sample to show how to manage deployment size in enterprise angular applications. I am sure it adds a great item to your list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,7 +168,6 @@ Table of contents:
 * [Nir kaufman](https://twitter.com/nirkaufman)
 * [Dmitriy Shekhovtsov](https://twitter.com/valorkin)
 * [Jeff Delaney](https://twitter.com/jeffdelaney23)
-* [**{{** add_expert **}}**](https://github.com/gdi2290/awesome-angular/edit/gh-pages/README.md)
```
Comment (rolandjitsu) [inline README.md:171]: Why was this removed?

**Comment by unosbaghaie** [comment]
The build failed and the reason was that this file wasn't available .

File: README.md (hunk 1 above)
Comment (unosbaghaie) [reply README.md]: The build failed and the reason was that this file wasn't available .

Comment (younos1986) [reply README.md:171]: The build failed, because this file wasn't available .

Comment (rolandjitsu) [reply README.md:171]: There's probably lots of broken links. You can add it to the whitelist to ignore it for now.

Comment (younos1986) [reply README.md:171]: Aha , Thanks for the info

Comment (younos1986) [reply README.md:171]: Should I do something else?
Would you please explain how to add broken links to whitelist to ignore?

Comment (rolandjitsu) [reply README.md:171]: See [validate_urls.sh](https://github.com/PatrickJS/awesome-angular/blob/gh-pages/scripts/validate_urls.sh).

**Comment by rolandjitsu** [comment]
Closing as no activity.
